plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-child.c

```diff
@@ -22,6 +22,17 @@ fu_logitech_bulkcontroller_child_write_firmware(FuDevice *device,
 						GError **error)
 {
 	FuDevice *proxy = fu_device_get_proxy(device);
+	/*
+	 * need to reopen the device, as at composite_cleanup time, parent device gets closed
+	 *
+	 * set the flag, to let parent know that firmware update is for child, no need to wait for
+	 * replug event, after child firmware is updated
+	 *
+	 */
+	if (!fu_device_open(proxy, error))
+		return FALSE;
```

Comment:
rather than doing this; can you set FU_DEVICE_PRIVATE_FLAG_USE_PROXY_FOR_OPEN in the child instance?

Reply:
Fixed. Thanks.